src/App.js

```diff
@@ -40,6 +40,7 @@ class App extends Component {
     //They dispatch the most current API call to the Redux store
     this.props.dispatch(countyFetch());
     this.props.dispatch(zipFetch());
+    console.log('zoom:', this )
```

Comment:
Just making sure this console.log is still necessary!

Reply:
I have removed it. I had added it to keep track of some props during coding.